csharp.html.markdown

```diff
@@ -18,13 +19,15 @@ C# is an elegant and type-safe object-oriented language that enables developers
 Multi-line comments look like this
 */
 /// <summary>
-/// This is an XML documentation comment
+/// This is an XML documentation comment which can be used to generate external
+/// documentation or provide context help within an IDE
 /// </summary>
+//public void MethodOrClassOrOtherWithParsableHelp() {}
 
 // Specify namespaces application will be using
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
+using System.Data.Entity; // Add dll reference with NuGet: Install-Package EntityFramework
```

Comment:
All other `usings` are available right away but System.Data.Entity needs a download first. So if someone loaded this in Visual Studio, there would be a compile error. That is until one does "Install-Package EntityFramework" in the NuGet Console.

Perhaps I should add a tidbit about NuGet? (=pip/npm/... for .NET)
Or explain why `System.Data.Entity` is not available out-of-the-box?